lib/src/supabase.dart

```diff
@@ -179,10 +48,13 @@ class Supabase {
     );
     if (url != null && anonKey != null) {
       _instance._init(url, anonKey);
-      _instance._authCallbackUrlHostname = authCallbackUrlHostname;
       _instance._debugEnable = debug ?? kDebugMode;
-      _instance._localStorage = localStorage ?? const LocalStorage();
       _instance.log('***** Supabase init completed $_instance');
+
+      SupabaseAuth.initialize(
```

Comment:
Do you think without await here, it can cause problem when user try to access session/user after calling `Supabase.initialize(...)`

Reply:
indeed

Reply:
@phamhieu done